index.d.ts

```diff
@@ -1,3 +1,16 @@
+interface Options {
+	/**
+	Optional filter function. function should return false to prevent the module from being removed from cache.
+
+	Use case: To prevent any module in node_modules from being uncached.
+
+	`clearModule('./my-module', { filter: name => !name.match(/node_modules/) })`
```

Comment:
This is not how you do a code block in docs.

Reply:
this is, if its a one liner. however it can be changed if needed